crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -83,6 +84,23 @@ impl ZombienetCommand {
 			.to_string();
 			log::warning(format!("⚠️ The following binaries specified in the network configuration file cannot be found locally:\n   {list}"))?;
 
+			// Check for any local binaries which need to be built manually
+			let local: Vec<_> = missing.iter().filter(|(_, _, local)| *local).collect();
+			if local.len() > 0 {
+
+				for (name, binary, _) in local.iter() {
+					intro(format!("{}: Building your parachain", style(" Pop CLI ")
```

Comment:
To keep consistency with the funcionality we have in Contracts, can you add a warning message too? like the one you added for contracts?
`log::warning(format!("NOTE: contract has not yet been built."))?;`